### PR TITLE
Implementation of robust Optimal Design of Experiments (ODoE)

### PR DESCRIPTION
**Pull request by sotorrio1**
Adding the latest feature to the SDoE module. User will need psuade-lite to use it.

### CODE REVIEW

**Review by candcook — DISMISSED** [review]
This is looking good.

**Comment by ksbeattie** [comment]
@sotorrio1, should we wait until psuade-lite is ready and open to merge this, or can we safely do it before without requiring psuade-lite that isn't yet available?

**Review by candcook — APPROVED** [review]
Looking good

**Pull request merged**
